src/style.css

```diff
@@ -1,3 +1,90 @@
+html {
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+
+  --link-color: #00acdc;
+}
+
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  text-decoration: none;
+  margin-right: 20px;
+  align-items: center;
+}
+
+.nav__link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  display: flex;
+  font-weight: 500;
+  padding: 23px 0;
+}
+
+.nav__link:hover {
+  color: var(--link-color);
+}
+
+.is-active {
+  position: relative;
+  color: var(--link-color);
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--link-color);
+  border-radius: 8px;
+}
+
+.is-active:hover::after {
+  width: 100%;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link[data-qa='hover'] {
+  color: var(--link-color);
 }
```

Comment:
```suggestion

```

Reply:
But if I delete these lines, some of the conditions of the job will not be fulfilled, can you please tell me why it is necessary to delete these lines ?